src/types/tau-prolog/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export namespace type {
```

Comment:
I am not quiet sure if it is properly done like this. There is now an linter error on the import in prologSession.

Reply:
You could import `type.Session` there directly, can't you?